From 488dc6dfa535206cf63c1c8cfefe972bee059c6f Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@oracle.com>
Date: Thu, 2 Aug 2012 17:55:47 -0700
Subject: [PATCH:cf] On Solaris 11 & later, use /usr/gnu/bin/install for
 InstallCmd

Older Solaris releases use /usr/ucb/install from the SunOS 4 compatibility
packages, but the EOL of that package has been announced now that it has
been 20 years since the SunOS 4.x -> Solaris 2.x transition.

Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
---
 sun.cf |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/sun.cf b/sun.cf
index 516a586..3f93d03 100644
--- a/sun.cf
+++ b/sun.cf
@@ -429,7 +429,9 @@ XCOMM SunPro C++ CplusplusCompilerMajorVersion./**/CplusplusCompilerMinorVersion
 #   define XawI18nDefines    -DHAS_WCHAR_H -DHAS_ISW_FUNCS
 #  endif
 # endif
-# if OSMinorVersion > 3
+# if OSMinorVersion > 10
+#  define InstallCmd	    /usr/gnu/bin/install
+# elif OSMinorVersion > 3
 #  define InstallCmd	    /usr/ucb/install
 # endif
 #ifndef LargefileDefines
-- 
1.7.9.2

